container_main.py

```
@@ -78,7 +78,9 @@ def run_api_server():
         from api_server import app
 
         port = int(os.environ.get("PORT", 5000))
-        app.run(host="0.0.0.0", port=port, debug=False, use_reloader=False)
+        app.run(
+            host="0.0.0.0", port=port, debug=False, use_reloader=False
```

Ruff (0.15.2)

[error] Possible binding to all interfaces

(S104)